lib/serialisers/json-0.6.js

```diff
@@ -82,10 +82,14 @@ module.exports = createClass({
     // Wrap default in array
     var defaultValue = attributes.get('default');
     if (defaultValue && defaultValue.content) {
+      defaultValue.content.attributes.remove('typeAttributes');
```

Comment:
Are you sure you want to remove the whole of typeAttributes instead of just fixed?

Reply:
You'd never have required, optional or any other typeAttribute in this place in 0.6? I was thinking perhaps it wasn't worth the additional logic as it shouldn't happen. But we could alter it if you think it might happen.

Reply:
Hmm.. I've thought long and hard about this. I am proposing to leave it as it is.